### PR TITLE
Proposal for a Dockerfile for OpenJ9

### PR DESCRIPTION
**Pull request by jams008**
I add OpenJ9 in Dockerfile-OpenJ9 I adjust to the requirements on the issue.

jobs:

- [x] Making Dockerfile OpenJ9 for Jenkins

- [x] adjust -Xshareclasses -Xscmx

test:

- [x] standard test running Jenkins

note:
I have not done in-depth testing of these images. You can do in-depth testing and review.

### CODE REVIEW

**Comment by dsouzai** [comment]
Looks like @kamiyaa also has a change for this (https://github.com/kamiyaa/docker/commit/28f437e7a073ea94eec668084a7a5fa1dda4f475) which includes changes to the Makefile. Perhaps you two could coordinate?

File: jenkins-openj9.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#! /bin/bash -e
```
Comment (oleg-nenashev) [inline jenkins-openj9.sh]: Any chance we could avoid introducing a new script?

Comment (jamaluddinfikri) [reply jenkins-openj9.sh]: in the new script I just added this '-Xshareclasses -Xscmx'

Comment (jamaluddinfikri) [reply jenkins-openj9.sh]: but if you don't need the cache '-Xshareclasses -Xscmx' actually don't have to use this jenkins-openj9.sh

Comment (olblak) [reply jenkins-openj9.sh]: Can't you just add those to JAVA_OPTS by default?

Comment (jamaluddinfikri) [reply jenkins-openj9.sh]: I have not tried it

**Comment by slide** [comment]
We have an openj9 image now, closing this out as superseded by #890

**Pull request closed without merging**